Dockerfile

```
@@ -18,20 +18,33 @@ RUN set -ex \
 ###############################################################################
 
 FROM alpine:3.7
+
+# Create a group and user
+RUN addgroup -g 101 orchestrator && adduser -u 101 -g 101 -S orchestrator
```

Use 999 uid/gid just to be safe.

```suggestion
RUN addgroup -g 999 orchestrator && adduser -u 999 -g 999 -S orchestrator
```